draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1026,21 +1030,22 @@ document's "site for cookies".
 
 Shared workers may be bound to multiple documents at once. As it is quite
 possible for those documents to have distinct "site for cookie" values, the
-worker's "site for cookies" will be the empty string in cases where the values
-diverge, and the shared value in cases where the values agree.
+worker's "site for cookies" will be an origin set to a globally unique 
+identifier in cases where the values diverge, and the shared origin in cases 
+where the values agree.
 
 Given a WorkerGlobalScope (`worker`), the following algorithm returns its "site
-for cookies" (either a registrable domain, or the empty string):
+for cookies":
 
-1.  Let `site` be `worker`'s origin's host's registrable domain.
+1.  Let `site` be `worker`'s origin.
 
 2.  For each `document` in `worker`'s Documents:
 
     1.  Let `document-site` be `document`'s "site for cookies" (as defined
         in {{document-requests}}).
 
-    2.  If `document-site` is not an exact match for `site`, return the empty
-        string.
+    2.  If `document-site` is cross-site with `site`, return an origin
```

Comment:
Don't think we actually ever defined what it means for two origins to be cross-site, so maybe say "not same-site with". (Or define cross-site on line 942)